test/jinx_test.exs

```diff
@@ -1,8 +1,49 @@
 defmodule JinxTest do
   use ExUnit.Case
-  doctest Jinx
+  import Jinx, only: [json_for: 2, json_for: 3]
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  describe "json_for" do
```

Comment:
Are you supposed to split varying arity in the describe? That isn't the pattern we have followed elsewhere.